src/cljs/ataru/virkailija/application/attachments/virkailija_attachment_view.cljs

```diff
@@ -54,24 +59,33 @@
   (str "/lomake-editori/api/files/content/" (:key attachment)))
 
 (defn- attachment-skimming-filename [selected-attachment]
-  (let [download-label    (str "lataa ("
-                               (-> selected-attachment :size u/size-bytes->str)
-                               ")")
-        download-url      (download-url selected-attachment)
-        filename          (-> selected-attachment :filename)
-        can-display-file? @(re-frame/subscribe [:virkailija-attachments/can-display-file? (:key selected-attachment)])]
+  (let [download-label     (str (cu/get-virkailija-translation :load-attachment-in-skimming)
+                                " ("
+                                (-> selected-attachment :size u/size-bytes->str)
+                                ")")
```

Comment:
Tässä myös format ehkä voisi olla selkeämpi:

`(format "%s (%s)" (cu/get-virkailija-translation :load-attachment-in-skimming) (-> selected-attachment :size u/size-bytes->str))`